src/app/views/discover/home/_banner/banner.vue

```diff
@@ -62,6 +62,33 @@
 									v-app-track-event="`home:banner:follow-${item.game.id}`"
 								/>
 							</div>
+							<div v-else-if="item.community">
+								<app-button
+									v-if="!item.custom_url"
+									solid
+									:to="location"
+									v-app-track-event="`home:banner:community-${item.community.path}`"
+								>
+									<translate>View Community</translate>
+								</app-button>
+								<app-button
+									v-else-if="item.custom_url"
+									solid
+									:href="item.custom_url"
+									target="_blank"
+									v-app-track-event="`home:banner:custom-community-${item.community.path}`"
+								>
+									{{ item.custom_text }}
+								</app-button>
+
+								<app-community-join-widget
+									v-if="shouldShowJoin"
```

Comment:
maybe instead of not showing for guests, use v-app-auth-required on this?

Reply:
Idk, this is following what the game follow button does. Should we do it for both?